docs/contributing.md

```diff
@@ -19,17 +19,28 @@ git clone git@github.com:infinitered/reactotron.git
 cd reactotron
 ```
 
-Install `lerna` and run the setup script to install the dependencies & run tests.
+If you don't already have `lerna`, install it:
+
+```
+npm install --global lerna
```

Comment:
You won't need to install this globally.  This should come in as a local dependency because it's in the `package.json` at the root of the project.  I'm thinking we remove this part.